_posts/2020-11-25-#19858.md

```diff
@@ -10,13 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+- An [eclipse attack](https://bitcoinops.org/en/topics/eclipse-attacks/) is
+  when an adversary is able to isolate a victim from the bitcoin network by
+  overtaking all of its connections.
+
+- Block-relay-only connections are a type of connection where two nodes do not
```

Comment:
maybe omit "two" unless additional info is added, as it's not clear from the preceding text where the number comes from and this sententce describes what block-relay-only connections are and not the implementation

Reply:
done